src/openforms/exception_handler/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from .views import ErrorDetailView
+
+app_name = "exception_handler"
+
+urlpatterns = [
+    path("fouten/<exception_class>/", ErrorDetailView.as_view(), name="error-detail"),
+]
```

Comment:
Please put all of the code in this package/app into the `openforms.api` package.

The URL route can be included directly from `openforms.urls` so we don't get the `/api` prefix for the human-facing views.